src/ReactLeafletMap/layers/CanvasVectorLayer/shapes/shapes.js

```diff
@@ -0,0 +1,39 @@
+import helpers from "./helpers";
+
+/**
+ * Draw shape to the given canvas
+ * @param context {Object} canvas context
+ * @param center {Object} center point of the shape
+ * @param style {Object} Panther style definition
+ * @param pixelSizeInMeters {number | null}
+ */
+function draw(context, center, style, pixelSizeInMeters) {
+
+	// TODO add other shapes
+	if (style.shape === "square") {
+		square(context, center, style, pixelSizeInMeters);
+	} else {
+		circle(context, center, style, pixelSizeInMeters);
+	}
+}
+
+function square(context, center, style, pixelSizeInMeters) {
+	const size = helpers.getSize(style.size, pixelSizeInMeters);
+	const a = 2*size; // side length
```

Comment:
Size is specified in style as style.size (it is in definition at the same level as fill, outlineWidth etc.). Currently, size=radius for circles and 2*size=side for squares. And yes, squares will appear larger than circles. We can, for instance, use size as radius -> calculate volume of circle -> derive size for squares. But I'm affraid that triangles, for example, will still appear larger...

Reply:
I'd expect size to be roughly diameter rather than radius. And yes, all shapes (except the base one) might need adjustment. Just like we did it in the first insar. (Can't find that right now)

Reply:
(Ah, never mind first insar. Found it, we did it manually there. So who knows where we used the adjustment, but it's not really important anyway.)

Reply:
Ok, we need to discuss (and try) this deeper, apparently.